src/store/reducers/appReducer.js

```diff
@@ -121,7 +122,29 @@ const initialState = {
   isDefaultLogo: true,
   azureMapsKey: '',
   deviceGroupFlyoutIsOpen: false,
-  timeInterval: 'PT1H'
+  timeInterval: 'PT1H',
+
+  //TODO: Get this from the server. This is just hardcoded test data for now.
+  userPermissions: new Set([
+    permissions.createDeviceGroups,
```

Comment:
Because when I'm testing a specific permission, it is convenient to simply comment out the line granting that permission.  This will not be the final code.... When Jill's server side code is complete, we'll get the list from that and not a hard coded list nor the values from the constant.